### PR TITLE
Adding Additional Party Types

### PR DESCRIPTION
**Pull request by cworsnop-figure**
## Description

- Adding additional party types (CONTROLLER, VALIDATOR) for DART usecases 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [x] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: docs/proto-docs.md [hunk 1]
```diff
@@ -3513,6 +3515,8 @@ A Timestamp represents a point in time using values relative to the epoch.
 | PARTY_TYPE_OMNIBUS | 7 |  |
 | PARTY_TYPE_PROVENANCE | 8 |  |
 | PARTY_TYPE_MARKER | 9 |  |
+| PARTY_TYPE_CONTROLLER | 10 |  |
+| PARTY_TYPE_VALIDATOR | 11 |  |
```
Comment (dwedul-figure) [inline docs/proto-docs.md:3522]: 💯 for updating the docs!

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,7 +37,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
-None
+### Features 
+
+  * Add CONTROLLER, and VALIDATOR PartyTypes for contract execution. 
+
+### Improvements
```
Comment (dwedul-figure) [inline CHANGELOG.md:44]: If there's something else that needs to be changed, maybe also remove this `### Improvements` line. It's not hurting anything, though, and we'll most likely need it in the long run, so no need to remove it unless there's other changes to push up too.

**Review by dwedul-figure — APPROVED** [review]
LGTM.

**Review by iramiller — DISMISSED** [review]
LGTM. Nice. Don't worry about the change log item... we have a lot of stuff going in there and that file gets a lot of work as we go.

**Review by derekadams — DISMISSED** [review]

**Comment by cworsnop-figure** [comment]
Pushed a new commit which is signed... so need new reviews. :sorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorrysorry:

**Review by iramiller — APPROVED** [review]

**Review by egaxhaj-figure — APPROVED** [review]
LGTM

**Pull request merged**